src/py21cmfast/outputs.py

```diff
@@ -227,6 +227,7 @@ def _init_arrays(self):
         )
         self.z_re_box = np.zeros(self.user_params.HII_tot_num_pixels, dtype=np.float32)
         self.dNrec_box = np.zeros(self.user_params.HII_tot_num_pixels, dtype=np.float32)
+        self.TkIGM_box = np.zeros(self.user_params.HII_tot_num_pixels, dtype=np.float32)
```

Comment:
In Python, a more descriptive name is better (but you don't need to append the `_box`).

Reply:
just to add on the naming convention: all the fields are "IGM" and "boxes"..  no need for this.  Tgas or Temperature or Tk should be fine...

Reply:
Yeah, I originally felt that I didn't want to change too many of the names, so left most of these quantities with an appended `_box`, even though I feel like that's a bad name python-wise. Probably in v4 we'll try switch to better names for everything. But let's try to start with better names for new variables now :-)

Reply:
I don't think `Tk` is particularly descriptive. `temp_gas`, `temp_igm`, or even `temp_gas_kinetic` or `temp_kinetic` (I presume that's what the `k` stands for?) would be better.

Reply:
I'm struggling with the naming. It's just both Tk and TkIGM are kinetic temperatures of the gas. It's just one is in SpinBox and one is in IonizedBox... With the former being the temperature of HI while the latter is all gas... 

But I'm a bit worried of renaming TkIGM IonizedBox to Tk and Tk in SpinBox to Tk_HI. I think people who are used to the previous naming convetion might get confused with what's going on with Tk...

Reply:
Why don't you call `TkIGM` something like `temp_kinetic_all_gas` or `temp_kinetic_igm`?

Reply:
and keep Tk as the way it is?

Reply:
Yup, for now. Since it already exists in the code, and we want to try to not break the API as much as possible from here on in until 3.0 is released. That will have to be changed on v4.0

Reply:
okay, I renamed TkIGM to temp_kinetic_all_gas and wrote a note in the CHANGELOG about Tk and temp_kinetic_all_gas